n_query: 5
n_repetitions: 100
kind_test: scheletro
start_query: 0

  # In Degree
query_0:
  start_configuration:  [-17.3, -125.3,-74.92, -68.79, -93.59, -42.39]
  stop_configuration: [-95, -125.3,-74.92, -68.79, -93.59, -42.39]
query_1:
  start_configuration:  [-36.75,-165.98,-60.15,-133.53,-61.48,-31.12]
  stop_configuration: [-41.6,-92,-112.97,-63.25,-89.27,-18.53]
query_2:
  start_configuration:  [-66.9,-164.41,-93.43,-96.24,-68.62,-6.63]
  stop_configuration: [-5.36,-111.33,-93.48,-68.88,-92.5,-6.48]
query_3:
  start_configuration:  [-31,-158,-81,-52,66,84]
  stop_configuration: [-95,-141,-49,-52,-91,84]
query_4:
  start_configuration:  [-57,-60,-137,-63,-88,-76]
  stop_configuration: [-57,-151,-112,-84,-88,-76]


# 1) rosbag play skeleton_record.bag -l
# 2) roslaunch test_skeleton test_planning.launch
